src/raftstore/store/snap.rs

```diff
@@ -2398,7 +2398,7 @@ pub mod tests {
             .unwrap();
 
             // TODO: this size may change in different RocksDB version.
-            let snap_size = 1438;
+            let snap_size = 1658;
```

Comment:
em, why 1658 here?

Reply:
> em, why 1658 here?

@siddontang Ignore this please, it's just for the time being since I merged the branch to upgrade rocksdb. Otherwise it won't build for me on OS X.

Reply:
As the todo mentioned, different version of RocksDB has different size of header.